packages/adapters/src/adapters/Cornerstone3D/ArrowAnnotate.ts

```diff
@@ -11,6 +11,23 @@ const trackingIdentifierTextValue = `${CORNERSTONE_3D_TAG}:${ARROW_ANNOTATE}`;
 const { codeValues, CodingSchemeDesignator } = CodingScheme;
 
 class ArrowAnnotate {
+    static toolType = ARROW_ANNOTATE;
```

Comment:
These changes are just to make this into typescript - makes things cleaner generally, but no functional change.